furusiya/main_interface.py

```diff
@@ -33,13 +37,22 @@ def run(cls, to_run):
             panel=(SCREEN_WIDTH, PANEL_HEIGHT),
             fps_limit=LIMIT_FPS
         )
+        cls.saveload = SaveLoad(cls)
```

Comment:
This makes my brain hurt. Why isn't this `Game.saveload`?

Reply:
Because `cls` in this case refers to `Game`? Both do just as well.